src/libs/DeckBuilderForm.js

```diff
@@ -12,18 +14,27 @@ class DeckBuilderForm extends React.Component {
   constructor(props) {
     super(props);
 
-    // Get the right set of decks
-    let decks = this.prepareDecks(decksImport.se); // TODO: Add functionality to change decks here
 
-    // Set the state of this component
-    this.state = {
-      decks:          decks,                    // The current decks that the user has selected
+    //Set state of the form, decks is empty while we wait for the JSON to return with the list of sections.
+      this.state = {
+      decks:          [],                       // The current decks that the user has selected, an empty arraywhile we wait for the JSON to return
       indeterminate:  false,                    // For the indeterminate state of the "check all" checkbox
       checkAll:       false,                    // Whether all the checkboxes are checked or not
       checkedList:    [],                       // The list of checked checkboxes (decks)
       shouldNotify:   this.props.shouldNotify,  // Whether the checkbox for "notify" should be checked or not
       logo:           null                      // The current logo (determined from the customer name)
-    };
+      };
+
+
+    //Send an XMLHttpRequest to retreive the deck sections and lengths from: https://docs.google.com/spreadsheets/d/1lAorVfpa8xeOMuT95lLj_I8YblvzMzy2-TlH1ttkn-A/edit#gid=0
+    var xhr = new XMLHttpRequest()
```

Comment:
This xhr request should really be called from `componentDidMount` as `this.setState()` won't work before the component is mounted. As it's async it probably doesn't make any difference in practice, but could introduce issues in the future.